### ISSUE TITLE
Unable to render Map as json with Hibernate4Module

### ISSUE THREAD

**Issue by wenky** [FasterXML/jackson-datatype-hibernate#107]
Hi,
I'm unable to render json  from the Map.

is working fine without includingHibernat4Module in the ObjectMapper.

Could anyone help me in this scenario.

**Comment by wenky** [comment]
Hi,

Could anyone please respond to this...

**Comment by wenky** [comment]
FYI:I'm getting the following exception trace:
`java.lang.NullPointerException
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serializeFields(MapSerializer.java:472)
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serialize(MapSerializer.java:412)
	at com.fasterxml.jackson.databind.ser.std.MapSerializer.serialize(MapSerializer.java:27)
	at com.fasterxml.jackson.databind.ser.BeanPropertyWriter.serializeAsField(BeanPropertyWriter.java:505)
	at com.fasterxml.jackson.databind.ser.std.BeanSerializerBase.serializeFields(BeanSerializerBase.java:639)
	at com.fasterxml.jackson.databind.ser.BeanSerializer.serialize(BeanSerializer.java:152)
	at com.fasterxml.jackson.databind.ser.DefaultSerializerProvider.serializeValue(DefaultSerializerProvider.java:128)
	at com.fasterxml.jackson.databind.ObjectMapper.writeValue(ObjectMapper.java:1902)
	at org.springframework.http.converter.json.AbstractJackson2HttpMessageConverter.writeInternal(AbstractJackson2HttpMessageConverter.java:231)
	at org.springframework.http.converter.AbstractHttpMessageConverter.write(AbstractHttpMessageConverter.java:208)
	at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.writeWithMessageConverters(AbstractMessageConverterMethodProcessor.java:161)
	at org.springframework.web.servlet.mvc.method.annotation.AbstractMessageConverterMethodProcessor.writeWithMessageConverters(AbstractMessageConverterMethodProcessor.java:101)
	at org.springframework.web.servlet.mvc.method.annotation.RequestResponseBodyMethodProcessor.handleReturnValue(RequestResponseBodyMethodProcessor.java:167)
	at org.springframework.web.method.support.HandlerMethodReturnValueHandlerComposite.handleReturnValue(HandlerMethodReturnValueHandlerComposite.java:71)
`

**Comment by cowtowncoder** [comment]
A reproduction would be needed, as well as basic information like version of Jackson used.
Use of spaces in property names should be fine for JSON, although could pose problems if outputting XML via Jackson xml module.
I do not have any good guesses as to what might be going wrong here unfortunately.

**Comment by wenky** [comment]
Hi,
We  hav added this bean in spring boot.
Without Hibernate4Module,the controller(below) able to render the json in the response.
With Hibernate4Module,it is giving me an error page and when I debug the code,I get the above stacktrace that may help you in your investigation.
```
public MappingJackson2HttpMessageConverter jacksonMessageConverter() {
		MappingJackson2HttpMessageConverter messageConverter = new  MappingJackson2HttpMessageConverter();
		ObjectMapper mapper = new ObjectMapper();
		mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES,false);
		
		mapper.registerModules(new Hibernate4Module());	
                    //mapper.registerModules(new SimpleModule());		
               messageConverter.setObjectMapper(mapper);
		return messageConverter;

	}
```

And this is my Controller.
```
 @ResponseBody
@RequestMapping(value={"/test"},method = RequestMethod.GET)
    public Map test(Model model){
        Map m=new HashMap<>();
        m.put("name","Wenky");
        m.put("company","Thomson Reuters");
        return m;
    }
```

**Comment by wenky** [comment]
Hi All:

Please ignore this.I got it fixed by using 2.7.4 version
